.travis.yml

```diff
@@ -1,42 +1,32 @@
 language: python
 sudo: false
-cache:
-- pip
+dist: trusty
+cache: pip
 python:
-- '2.7'
-- '3.4'
-- '3.5'
-- pypy
```

Comment:
Django will actually support Python 2.7 throughout Django 1.11 LTS which should be supported until (at least) April 2020. So all nice Django package authors are stuck with supporting that, too, if we want to be human to people who haven't migrated to Python 3 yet. So, PyPy doesn't really cost anything to support, and the PyPy project might well get PyPy working with Python >= 3.5 well before 2020. They have also talked a lot about supporting Python 2 and Python 3 with the same code base (and the same JIT).

I, too, use PyPy with some projects and it just always sucks when package support is dropped when it doesn't cost anything to keep - if you support Python 2.7, supporting PyPy is always no-effort with existing test infrastructure.